graphql/schema.graphql

```diff
@@ -195,7 +195,19 @@ type Dependency {
   buildVariant: String!
 }
 
+type PatchMetadata {
```

Comment:
Revision and Order are available on the base task model, so I think this just needs to be the single author field